### PR TITLE
Stop using memory queue

### PR DESCRIPTION
**Pull request by b2mdevelopment**

### CODE REVIEW

File: src/reyna/reyna.csproj [hunk 1]
```diff
@@ -17,7 +17,7 @@
     <PlatformFamilyName>WindowsCE</PlatformFamilyName>
     <PlatformID>E2BECB1F-8C8C-41ba-B736-9BE7D946A398</PlatformID>
     <OSVersion>5.00</OSVersion>
-    <DeployDirSuffix>reyna</DeployDirSuffix>
+    <DeployDirSuffix>elemez</DeployDirSuffix>
```
Comment (b2mdevelopment) [inline src/reyna/reyna.csproj]: What is this? @CharlesGarth

Comment (CharlesGarth) [inline src/reyna/reyna.csproj:20]: What is this? :D

Comment (charlierobson) [inline src/reyna/reyna.csproj:20]: "The node is only there for Smart Device projects.
It sets the directory when deploying to a device via USB (or other means). You can find this setting via project properties, Device menu, Output file folder field."

Could be left blank.

**Comment by charlierobson** [comment]
Simpler :+1:

File: src/reyna/reyna.csproj (hunk 1 above)
Comment (b2mdevelopment) [inline src/reyna/reyna.csproj:20]: This the target deployment directory.

**Pull request merged**